---
title: Research in the Microbial Pangenomes Lab
layout: default
group: research
---

## We are interested in microbial pangenomes

We are perhaps used to think of genetic differences in the context of human genetics; two individuals differ on average in [about 5M sites in the genome](https://pubmed.ncbi.nlm.nih.gov/26432245/), or about 0.8% of the whole genome. When thinking about bacteria, genetic variation takes a whole different scale: two *E. coli* strains may differ for up to 60% of their gene content, not even counting for [SNPs](https://en.wikipedia.org/wiki/Single-nucleotide_polymorphism). This implies that each bacterial species does not have a single genome, but rather a diverse ensemble of gene combinations, collectively termed as a [pangenome](https://pubmed.ncbi.nlm.nih.gov/19086349/). The influence of this large genetic diversity on bacterial phenotypes is the main interest of the lab.

### Genotype-to-phenotype models in bacteria

<div class="row">
<div class="col-md-8">
Understanding how variations in genotype translate to variation in phenotype is a long-standing problem in molecular biology. We are interested in improving our ability to predict the phenotypic consequences of genetic variants in bacteria. This can be done with a combination of ["forward" mechanistic models](https://pubmed.ncbi.nlm.nih.gov/29280730/) or [more agnostic "reverse" statistical genomics approaches](https://pubmed.ncbi.nlm.nih.gov/30535304/). The lab has access to [a large collection of *E. coli* strains](https://evocellnet.github.io/ecoref/) for this exact purpose.
</div>
<div class="col-md-4">
<img class="img-fluid" src="/static/img/geno_to_pheno.jpg" alt="Genotype to phenotype scheme">
</div>
</div>

### Assigning function to the accessory genome

<div class="row">
<div class="col-md-10">
Certain bacterial species such as *E. coli* have so-called open pangenomes, meaning that new genes - also called "accessory genes" - will be discovered as the genomes of new isolates is being sequenced. Current functional genomics techniques simply cannot scale up fast enough to catch up with the deluge of genes of unknown function that are discovered every day. We are therefore interested in developing methods to assign function to accessory genes *in-silico*. The wealth of data available for reference species together with advancements in machine learning are a potential way forward to solve this problem.
</div>
<div class="col-md-2">
<img class="img-fluid" src="/static/img/accessory_genome.jpg" alt="Functional prediction for the accessory genome">
</div>
</div>

### Influence of bacterial pangenomes on antimicrobial resistance evolution

<div class="row">
<div class="col-md-10">
It has been demonstrated that [genetic variability across strains influences gene essentiality profiles](http://www.ncbi.nlm.nih.gov/pubmed/31885205). We suspect that these differences might also influence the ability to adapt to a selective pressure, such as the use of antimicrobials. Understanding whether different genetic backgrounds are more or less prone to develop antimicrobial resistance is therefore one of the ways the lab is trying to understand the influence of bacterial pangenomes to clinically relevant phenotypes.
</div>
<div class="col-md-2">
<img class="img-fluid" src="/static/img/amr.jpg" alt="Antimicrobial resistance evolution across genetic backgrounds">
</div>
</div>
