---
title: PhD position in bacterial genomics and machine learning available
author: Marco Galardini
short: mgalardini
layout: post
group: news
---

After a rather long hiatus we are hiring again!

We are looking for a candidate to fill a **computational PhD student position**, 
with the main task of better understanding which genetic elements ("genes")
make bacterial pathogens such as *E. coli*, *K. pneumoniae*, *P. aeruginosa*, and *E. faecium*
virulent and resistant to antibiotics. We have in fact developed methods ([here](https://pyseer.readthedocs.io)
and [here](https://www.microbial-pangenomes-lab.org/2024/07/10/Making-microbial-GWAS-accessible-through-a-bit-of-plumbing/))
to sift through large numbers of bacterial genomes for this exact purpose (see [here](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1010842),
[here](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1010112),
and [here](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1009065)), and it will be the job
of the candidate to use and improve these methods so that they can be applied to
an even larger number of genomes. We are also particularly interested in implementing
more machine learning methods and to integrate molecular phenotypes such as gene expression
and proteomics.

Our lab is part of the [RESIST excellence cluster](https://www.resist-cluster.de/en/) at
[Hannover Medical School (MHH)](https://www.mhh.de/presse-news/mhh-kann-antrag-auf-exzellenzuni-stellen),
and we are part of a collaborative project with [Susanne HÃ¤ussler](https://www.helmholtz-hzi.de/en/research/research-groups/details/molecular-bacteriology/)
and [Meike Stiesch](https://www.mhh.de/meike-stiesch) to identify genetic determinants of pathogenicity
and virulence in life-threatening bacterial infections.

We are looking for a candidate with a strong computational background and relevant technical skills:

* **Strong background** in computational biology, bioinformatics, computer science, or a related field.
* **Experience with programming languages** such as **Python** and workflow management systems such as **Snakemake**.
* **Familiarity with machine learning** techniques and libraries (e.g., **scikit-learn**, **TensorFlow**, **PyTorch**).
* **Knowledge of bacterial genomics** and related bioinformatic tools.
* Experience with **high-performance computing** (HPC).
* Experience with **version control systems** (e.g., **git**) and software development best practices.
* Experience with **statistical analysis** and **data visualization**.

We offer a fully-funded PhD position for a little longer than 3 years, to begin in February 2026.
The student will be enrolled in the [Biomedas graduate program](https://translationsallianz.de/en/train-biomedas/),
which offers curriculum specifically designed for
computational biology students.
The lab is located at [Twincore](https://twincore.de/en), embedded in a large scientific campus that include [Hannover Medical School](https://www.mhh.de/en),
and the [Center for Individualized Infection Medicine (CiiM)](https://www.ciim-hannover.de/en/); we are also part of the
[Helmholtz Centre for Infection Research (HZI)](https://www.helmholtz-hzi.de/en/), which offers lots of opportunities
for collaboration.

Hannover is the capital of Lower Saxony, in northern Germany, and is
an [affordable](https://www.numbeo.com/cost-of-living/compare_cities.jsp?country1=Germany&country2=Germany&city1=Munich&city2=Hanover&tracking=getDispatchComparison)
and [vibrant](https://www.visit-hannover.com) city.

Please apply as soon as possible by sending the following documents to **Marco Galardini** at [marco.galardini@twincore.de](mailto:marco.galardini@twincore.de):

1.  Your **CV**
2.  A short **motivation letter**
3.  The link to a **software portfolio** (e.g., your GitHub account)
4.  Contact details of **two references**

<img class="img-fluid" src="{{ site.url }}/static/img/news/20251106_hannover.jpg" alt="Two photos from hannover">

