---
title: When the culprit is under your nose
author: Marco Galardini
short: mgalardini
layout: post
group: news
---

People familiar with the field of bacterial genomics have long been aware that microbial
genomes are densely packed with genes, and thus are depleted of so-called "junk DNA" (a term that has fallen out
of fashion by the way!).
As a result, the more abundant protein coding portions of these genomes get the most attention
from researchers aiming to find which genetic variants explain phenotypic variation among isolates.
Previous work has however already shown that bacterial non-coding regions are both highly diverse
and show signals of being evolutionarly constrained. We also knew that these regions influence
the expression of genes encoded directly downstream from them. We therefore hypothesized
that we could uncover statistical associations between genetic variants in non-coding regions
and gene expression variability across isolates.

The results of this work [have just been published as a preprint](https://www.biorxiv.org/content/10.1101/2025.06.10.658842v1),
a work that was led by [Bamu]({{ site.url }}/members/#bfufordamaris) during her time as a PhD student
in the lab. Bamu was the very first person brave enough to join the lab, and has manged to
work both in the dry- and wet-lab, a feat that not many people can achieve!

Bamu indeed found that it was possible to identify at least one genetic variant whose presence
was associated with gene expression changes in up to 39% of tested genes in two important
bacterial pathogens (*E. coli* and *P. aeruginosa*). Using the right way to represent the
complex genetic variation (*i.e.* [gene-centric k-mers]({{ site.url }}/news/2023/11/09/Multiple-haystacks-are-beter-than-one/))
allowed Bamu to capture the highest proportion of associations.

<img class="img-fluid" src="{{ site.url }}/static/img/news/20250610_bars.jpg" alt="Barplots indicating the main results from the association analysis">

Once we found these associations, the next task would be to validate some of them and to understand the actual mechanism
operating behind the scene. Here Bamu used a combination of *in-silico* and *in-vitro* approaches, which very clearly
indicated that no single mechanism would be sufficient to explain the observed associations.

The last part of the study was instead dedicated to the understanding of the role of non-coding
genetic variation to antimicrobial resistance. Again, Bamu used her dry- and wet-lab skills to show that
indeed there are non-coding variants in both species that are associated with antimicrobial resistance.
This leads us to conclude that these often neglected regions of the bacterial genome need to be
taken into account if we want to be eventually able to make the most out of bacterial genomes.
